dom-event-delegation/main.js

```diff
@@ -0,0 +1,14 @@
+const taskList = document.querySelector('.task-list');
+
+taskList.addEventListener('click', names);
+
+function names(event) {
+  console.log('event.target', event.target);
+  console.log('event.target.tagName', event.target.tagName);
+
+  if (event.target.matches('button')) {
```

Comment:
This works, but we had you use `tagName` on line 7, to show you that this is how you should be selecting it. Again this isn't wrong, but when we tell you to use a property name like above, I would try and use that one. We have you console log it for a reason.